examples/vweb_orm_jwt/src/v.mod

```diff
@@ -0,0 +1,7 @@
+Module {
+	name: 'v'
```

Comment:
That `v` name is misleading - the v.mod file can be empty, if it is used just for anchoring the module lookup. If it is not empty, it should have valid information in it. Perhaps just `name: 'vweb_orm_jwt'` instead?